languages/cpp/CMakeBuild.md

```diff
@@ -29,6 +29,16 @@ cmake .. -DNLOHMANN=/path/to/include/nlohmann -DBOOST=/path/to/include/boost -DT
 cmake --build .
 ```
 
+## IDE Support
+
+You may need to manually set the CMake `TARGET` variable for your IDE. For CLion, add the following to the CMake options
+settings:
+
+```bash
+# macOS example
+-DTARGET=../../target/release/libbitwarden_c.dylib
```

Comment:
All language wrappers i have seen does not use release target.
```suggestion
-DTARGET=../../target/debug/libbitwarden_c.dylib
```
If we decide to go with debug, the `## Example` in this file will also need an update